web/massastation/src/mirage/factories/store.ts

```diff
@@ -28,4 +28,10 @@ export const storeFactory = Factory.extend<MassaStoreModel>({
   os() {
     return 'Linux';
   },
+  iscompatible() {
+    return Math.random() < 0.3 ? false : true;
```

Comment:
For what you need a `random()` value here? 🤔

Reply:
to simulate a case where some plugins are compatible and others are not. It's not a must have i can remove it if you're not sure it is a good idea

Reply:
Not remove it! I'm just trying to understand what you're doing here, because i see a `0.3` value and I don't what is that.

Reply:
There is 30% chance that the plugin is not compatible, I chose it arbitrarily

Reply:
Okay but this value is send by the backend in production? Or it's just for the dev mode?

Reply:
no it's mirage only